utilities/blob_db/blob_log_format.h

```diff
@@ -151,10 +154,21 @@ class BlobLogRecord {
   char subtype_;
   Slice key_;
   Slice blob_;
-  char *key_buffer_;
-  uint64_t kbs_;
-  char *blob_buffer_;
-  uint64_t bbs_;
+  std::unique_ptr<char[]> key_buffer_;
+  size_t kbs_;
+  std::unique_ptr<char[]> blob_buffer_;
+  size_t bbs_;
```

Comment:
Why not use std::string for buffers? Much safer, well defined interface, works seemlessly with Slice, std::string_etc.